crowdsourcing/models.py

```diff
@@ -467,4 +468,24 @@ class ConversationRecipient(models.Model):
 class UserMessage(models.Model):
     message = models.ForeignKey(Message)
     user = models.ForeignKey(User)
-    deleted = models.BooleanField(default=False)
\ No newline at end of file
+    deleted = models.BooleanField(default=False)
+
+class File(models.Model):
```

Comment:
"File" seems like a common term and could be taken by other libraries, could we call it RequesterInputFile or InputCSVFile (then you can keep the parse function here)